Client/Frontend/Home/FxHomeRecentlySavedCollectionCell.swift

```diff
@@ -32,13 +32,15 @@ class FxHomeRecentlySavedCollectionCell: UICollectionViewCell {
     weak var homePanelDelegate: HomePanelDelegate?
     weak var libraryPanelDelegate: LibraryPanelDelegate?
     var profile: Profile?
+    static var recentlySavedItems = [RecentlySavedItem]()
```

Comment:
Lets not have static vars and access it directly, instead if you would like to do size calculation get the cell and access it as a variable.

A better approach would be to have an intermediary class / model to help with accessing saved items